### ISSUE TITLE
Missing alternative texts and labels for accessibility

### ISSUE THREAD

**Issue by Nukker1** [Limekiller/moodle-block_openai_chat#69]
There are missing alternative texts for the buttons "Open chat window," "Submit," and "New chat," as well as a missing label for the text input field "Ask a question...." As a result, these elements are not read by screen readers, limiting accessibility for users who rely on assistive technology.

**Comment by Limekiller** [comment]
Thanks for the report. I've added alt text to the button icons and an aria-label tag to the input element.
